hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -217,7 +220,21 @@ public XceiverClientReply sendCommand(
       ContainerCommandRequestProto request, List<DatanodeDetails> excludeDns)
       throws IOException {
     Preconditions.checkState(HddsUtils.isReadOnly(request));
-    return sendCommandWithRetry(request, excludeDns);
+    return sendCommandWithTraceIDAndRetry(request, excludeDns);
```

Comment:
shall we do this only when tracing is enabled?

Reply:
Unfortunately, I'm not aware of a switch to turn tracing off globally. That will be a much bigger change than the scope of this ticket.